sw/airborne/arch/stm32/subsystems/radio_control/spektrum_arch.c

```diff
@@ -47,7 +47,7 @@
 #define ONE_MHZ 1000000
 
 /* Number of low pulses sent to satellite receivers */
-#define MASTER_RECEIVER_PULSES 5
+#define MASTER_RECEIVER_PULSES 4     // RAB : Changed from 5 to 4 to enable the R100 satellite rx to bind successfully
```

Comment:
@gautierhattenberger this was a modification recommended by @esden when we were having problems binding the specktrum RX. I looked through the mod history for the file and didn't see any change by @esden. I think it's needed.